### DISCUSSION TITLE
How to set cookie in request manually?

### DISCUSSION THREAD

**Discussion by sasgosh** [Huachao/vscode-restclient#843] [General]
Hello,

How to set cookie in request manually?

**Comment by geminigeek** [comment]
hi,
i have the same question ?

**Comment by felix-manibo-aa** [comment]
This works on my end.
### GET Token
# @name login
GET {{testAuthURL}} HTTP/1.1
Content-type: application/json
Cookie: Cookie01=value01;
Cookie: Cookie02=value01;

@access_token = {{login.response.body.token}}
